src/core/Table/Table.tsx

```diff
@@ -322,31 +322,27 @@ export const Table = <
     <div
       ref={(element) => setOwnerDocument(element?.ownerDocument)}
       {...getTableProps({
-        className: cx('iui-tables-table', className),
+        className: cx('iui-table', className),
         style,
       })}
       {...ariaDataAttributes}
     >
       <div>
         {headerGroups.slice(1).map((headerGroup: HeaderGroup<T>) => {
           const headerGroupProps = headerGroup.getHeaderGroupProps({
-            className: 'iui-tables-row',
+            className: cx('iui-table-header', 'iui-row'),
           });
           return (
             <div {...headerGroupProps} key={headerGroupProps.key}>
               {headerGroup.headers.map((column) => {
                 const columnProps = column.getHeaderProps({
                   ...column.getSortByToggleProps(),
                   className: cx(
-                    'iui-tables-cell',
-                    'iui-tables-head',
-                    { 'iui-active-sort': column.isSorted },
+                    'iui-cell',
```

Comment:
I see that header cell always has `cursor: pointer` if it was always sortable but that is not always true.
![image](https://user-images.githubusercontent.com/36186912/121853956-394a6800-ccfa-11eb-8da0-2fe66d711fc7.png)
Created an issue: https://github.com/iTwin/iTwinUI/issues/151

Reply:
Fixed